themes-default/slim/src/components/new-show.vue

```diff
@@ -8,7 +8,7 @@
 
                     <form id="addShowForm" @submit.prevent="">
 
-                        <form-wizard ref="formwizard" class="formwizard" title="" subtitle="" @change="updateFormWizardStep" color="rgb(95, 95, 95)">
+                        <form-wizard ref="formwizard" class="formwizard" title="" subtitle="" @on-change="updateFormWizardStep" color="rgb(95, 95, 95)">
```

Comment:
this is not how you should use the events in Vue.js. We use the shorthands.
https://vuejs.org/v2/style-guide/#Directive-shorthands-strongly-recommended